openpype/hosts/houdini/api/pipeline.py

```diff
@@ -48,6 +52,10 @@ def __init__(self):
         self._op_events = {}
         self._has_been_setup = False
 
+    @property
+    def houdini_project_settings(self):
+        return HoudiniProjectSettings()
```

Comment:
@BigRoy 
Would it be still clear that it's `houdini_project_settings` if I removed the `houdini` prefix ? 
```suggestion
    # Current project houdini settings
    @property
    def project_settings(self):
        return HoudiniProjectSettings()
```

Reply:
No it wouldn't but I would also want to say, don't do this. It doesn't cache it .. it being a property makes it feel like would be cached and doesn't touch the database each time. This doesn't solve anything of it potentially being slow but just made it more complex code without any benefit.

If we were to cache the local project settings, then I think that could be nice if e.g. available on a dedicated host mixing instead of implementing it only in the Houdini side. Plus that'd be a separate PR.

I'd revert this change.